requirements.txt

```diff
@@ -5,6 +5,7 @@ Flask==3.0.2
 Flask-Migrate==4.0.5
 Flask-SQLAlchemy==3.1.1
 greenlet==3.0.3
+gunicorn==21.2.0
```

Comment:
So I tried running the `gunicorn` command we give in learn locally, which should work. Guess what doesn't automatically get loaded when starting the flask app through gunicorn? The `.env`. This of course isn't a problem for actual deployment, since we set real environment variables there. But it's disappointing all the same.

Should we resign ourselves to `.env` defeat and just go back to what we had? Running `load_dotenv` in the `__init__.py`?

Reply:
That's true. And technically, it's better to avoid running the load_dotenv when it's not needed. We could mention passing the config param on the command line (which is a very common pattern) to test it as well (to avoid introducing another file).